cs/test/coreNS10/CoreNS10.csproj

```diff
@@ -0,0 +1,77 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\build\internal\Common.Internal.props" />
+  <PropertyGroup>
+    <TargetFrameworks>net45;</TargetFrameworks>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>UnitTest</RootNamespace>
+    <AssemblyName>Bond.UnitTestCoreNS10</AssemblyName>
+    <IsCodedUITest>False</IsCodedUITest>
+    <TestProjectType>UnitTest</TestProjectType>
+    <HasNonportableVersion>true</HasNonportableVersion>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <!-- This project is used to test the netstandard1.0 implementation of Bond -->
+    <BondTargetFramework>netstandard1.0</BondTargetFramework>
+    <BondIOTargetFramework>netstandard1.3</BondIOTargetFramework>
+    <!-- We need to specify the output directory since the .bond files are on another subdirectory -->
+    <BondOutputDirectory>$(IntermediateOutputPath)</BondOutputDirectory>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <IntermediateOutputPath>$(IntermediateOutputPath)\Properties\</IntermediateOutputPath>
+    <OutputPath>$(OutputPath)\Properties\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Fields' ">
+    <IntermediateOutputPath>$(IntermediateOutputPath)\Fields\</IntermediateOutputPath>
+    <OutputPath>$(OutputPath)\Fields\</OutputPath>
+    <BondOptions>--fields --collection-interfaces</BondOptions>
+  </PropertyGroup>
+  <ItemGroup>
+    <BondImportDirectory Include="..\core\import dir with spaces" />
```

Comment:
Minor: should end with trailing slash

Reply:
I'm getting this error: Invalid project item: "..\core\import dir with spaces". A file item cannot end with a path separator.